python-package/xgboost/dask.py

```diff
@@ -866,6 +880,7 @@ def predict(client, model, data, missing=numpy.nan, **kwargs):
     '''
     _assert_dask_support()
     client = _xgb_get_client(client)
+    LOGGER.warning(kwargs)
```

Comment:
?

Reply:
ah! I forgot to push my latest commits yesterday